src/Data/Uniques/Special/New.lua

```diff
@@ -5,7 +5,91 @@
 data.uniques.new = {
 
 -- New
-
+[[
+Mahuxotl's Machination
+Steel Kite Shield
+Requires Level 46, 60 Str, 60 Int
+Implicits: 1
+Chance to Block: 26%
+Corrupted Soul
+Divine Flesh
+Eternal Youth
+Everlasting Sacrifice
+Immortal Ambition
+Vaal Pact
+]],[[
+Glimpse of Chaos
+Vaal Mask
+Requires Level 62, 79 Dex, 72 Int
+Implicits: 1
+Nearby Enemies are Crushed
+Can be modified while Corrupted
+32% increased maximum Life and reduced Fire Resistance
+32% increased maximum Mana and reduced Cold Resistance
+31% increased Global maximum Energy Shield and reduced Lightning Resistance
+Chaos Resistance is Zero
+Corrupted
+]],[[
+Atziri's Rule
+Judgement Staff
+Requires Level 68, 113 Str, 113 Int
+Implicits: 1
++20% Chance to Block Spell Damage while wielding a Staff
+Grants Level 20 Queen's Demand Skill
+Queen's Demand can Trigger Level 20 Flames of Judgement
+Queen's Demand can Trigger Level 20 Storm of Judgement
+Cannot be Stunned
+Damage cannot be Reflected
+]],[[
+Tecrod's Gaze
+Murderous Eye Jewel
+Requires Level 40
++10 to Strength
+20% increased Main Hand Critical Strike Chance per Murderous Eye Jewel affecting you, up to a maximum of 100%
++10% to Off Hand Critical Strike Multiplier per Murderous Eye Jewel affecting you, up to a maximum of +50%
+]],[[
+Cane of Kulemak
+Serpentine Staff
+Requires Level 68, 85 Str, 85 Int
+Implicits: 1
++20% Chance to Block Attack Damage while wielding a Staff
+61% increased Unveiled Modifier magnitudes
+]],[[
+The Scales of Justice
+Chiming Spirit Shield
+Requires Level 49, 118 Int
+Implicits: 1
+Chance to Block: 23%
+13% increased Spell Damage
+Has no Energy Shield
++54 to maximum Life
++46 to maximum Mana
+3 to 7 added Fire Damage per 100 of Maximum Life or Maximum Mana, whichever is lower
+Unaffected by Ignite or Shock if Maximum Life and Maximum Mana are within 500
+]],[[
+Ulaman's Gaze
+Searching Eye Jewel
```

Comment:
```suggestion
Searching Eye Jewel
Requires Level 40
```
Missing level requirement.